integration/imagetags/app.yaml

```diff
@@ -1,3 +1,10 @@
+apiVersion: v1
```

Comment:
i don't understand why we're copy and pasting the namespace into every app.yaml, rather than using https://github.com/windmilleng/tilt/blob/master/integration/namespace.yaml